Created error messages in certain areas and fixed back tracking bug.

jpakey99 opened the pull request:
1. fixed back tracking bug
2. Fixed open file location bug
3. added error message when you do not upload enough images
4. added error message when you do not select output destination (could be removed, just thought it would be weird if it did not exist)

keenanm500 commented:
Looks like some good changes, though could you include some more info about the bugs you fixed (like how to reproduce them and what the expected behavior is?)

jpakey99 commented:
@keenanm500:
**Back Tracking Bug:**
Upload images -> Select "Next" to Output Destination page -> Select Output Destination and press Next -> Complete the Specify Role Images page and press Next: Skip Optional Formating -> Once on the Target Patch Selection page, press the back button until you are back at the Upload Images page where you see no images uploaded.  This update keeps the images there.  The progress bar stays highlighted to the furthest page you went to and this seems like it should remain the case.  This is an issue the Sponsors pointed out and wanted to be fixed.

**Open File Location Bug:**
Go through all the steps of processing images.  After it is done, a box appears with different options including the "Open File Location" button, which is currently disabled.  Chrome will not allow you to perform the action and per my last PR that got deleted, I do not think we are in a position to develop a workaround, so I deleted the button and associated logic code for it. We could accomplish this by calling the backend (python) and opening up the file system; however, I am not sure if this is a Windows-only behavior. This is an issue the Sponsors pointed out and wanted to be fixed.

**Adding Error Message When You do not Upload Enough Images:**
Pretty self-explanatory.   Currently, the system does nothing when you upload less than 6 images and press the next button leaving users confused.  Added a pop-up box to alert users and explain what is expected. This is not an issue the Sponsors pointed out; however, I believe it increases the user experience, particularly for new users.

**Adding Error Message When You do not Select Output Destination:**
This might be an unintended consequence of the above change, but users can move past selecting an output destination with no error being thrown.  I feel like this should not be the case so I added a check.  Can check with Sponsors to confirm. This is not an issue the Sponsors pointed out; however, I believe it increases the user experience, particularly for new users.
